third_party/civetweb/tnt/CMakeLists.txt

```diff
@@ -32,3 +32,5 @@ target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 target_link_libraries(${TARGET} ${CMAKE_DL_LIBS})
 
 target_compile_options(${TARGET} PRIVATE $<$<PLATFORM_ID:Linux>:-fPIC>)
+
+install(TARGETS ${TARGET} ARCHIVE DESTINATION third_party/${DIST_DIR})
```

Comment:
Is there a reason we need a new `third_party` directory? Seems like we already put third party binaries inside `lib`, like `libsmol-v.a` and `libdracodec.a`.

Reply:
Excellent catch, fixing this thanks!